complexradio/dist/metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+    "author": {
+      "name": "Tyler",
+      "id": "351991098387988491"
+    },
+    "service": "Complex Radio",
+    "description": {
+      "en": "Complex Radio provides quality music to its listeners, we also always have applications open for new presenters!"
+    },
+    "url": "complexr.pw",
+    "version": "0.1",

```

Comment:
Please make sure that the version number follows [semantic versioning standards](https://semver.org/)